Source/DafnyCore/Options/DeveloperOptionBag.cs

```diff
@@ -5,13 +5,18 @@ namespace Microsoft.Dafny;
 
 public class DeveloperOptionBag {
 
+  public static readonly Option<bool> SpillTranslation = new("--spill-translation",
+    @"In case the Dafny source code is translated to another language, emit that translation.") {
+    IsHidden = true
+  };
+  
   public static readonly Option<bool> UseBaseFileName = new("--use-basename-for-filename",
```

Comment:
There are instances of the old spelling that ought to be changed concurrently (even if the old spelling still works):
grep -r -e '--useBaseNameForFileName' ../.. | grep -v Binary
../../Test/lit.site.cfg:'--useBaseNameForFileName',
../../Source/DafnyDriver/DafnyDriver.cs:      "--useBaseNameForFileName",
../../Source/DafnyCore/Options/DeveloperOptionBag.cs:  public static readonly Option<bool> UseBaseFileName = new("--useBaseNameForFileName",
../../Source/IntegrationTests/bin/Debug/net6.0/TestFiles/LitTests/LitTest/lit.site.cfg:'--useBaseNameForFileName',
../../Source/IntegrationTests/bin/Debug/net6.0/TestFiles/LitTests/LitTest/lit.site.cfg:config.substitutions.append( ('%resolveargs', "--useBaseNameForFileName" ) )
../../docs/check-examples:##              "dafny verify --useBaseNameForFileName $F" and checks 
../../docs/check-examples:##              "dafny resolve --useBaseNameForFileName $F" and checks
../../docs/check-examples:##              "dafny run --useBaseNameForFileName $F" and, 
../../docs/check-examples:          "$dafny" $com --useBaseNameForFileName $options $text > actual

Reply:
Seems like this grep was not done on this PR

Reply:
My mistake, and I was also looking a a stale list of changes...
Thanks for making this option consistently spelled.